### PR TITLE
Fix build-tools integtest eclipse classpath

### LINKED ISSUE #60615: Eclipse doesn't import spock
Importing Elasticsearch into Eclipse yields an error the looks like:
```
Groovy:unable to resolve class pock.lang.Specification       abstractGradleFuncTest.groovy   line 25
```

It's been like this for a couple of months. Sorry I didn't open this one earlier!

### PR DESCRIPTION
**Pull request by breskeby**
Fixes #60615

### CODE REVIEW

File: buildSrc/build.gradle [hunk 1]
```diff
@@ -177,6 +177,12 @@ if (project != rootProject) {
 
     integTestRuntimeOnly(project(":libs:elasticsearch-core"))
   }
+  
+  eclipse {
+	classpath {
+		plusConfigurations += [configurations.integTestRuntimeClasspath]
+	}
```
Comment (nik9000) [inline buildSrc/build.gradle:184]: Probably better to switch to spaces here so it renders well everywhere.

**Review by nik9000 — COMMENTED** [review]
This is better! I'm not getting errors about spock not being compatible with groovy 3. when I try to manually switch my project to groovy 3 I get other errors.

**Comment by breskeby** [comment]
> This is better! I'm not getting errors about spock not being compatible with groovy 3. when I try to manually switch my project to groovy 3 I get other errors.

that's expected as the groovy version we use (and gradle uses is 2.5.x

File: buildSrc/build.gradle (hunk 1 above)
Comment (breskeby) [reply buildSrc/build.gradle]: fixed

**Comment by nik9000** [comment]
> that's expected as the groovy version we use (and gradle uses is 2.5.x

We should probably fix the import then somehow. I'm not sure what is up with it.

**Review by nik9000 — APPROVED** [review]
LGTM

**Comment by breskeby** [comment]
@elasticmachine test this please

**Comment by breskeby** [comment]
> > that's expected as the groovy version we use (and gradle uses is 2.5.x
> 
> We should probably fix the import then somehow. I'm not sure what is up with it.

Can you elaborate what problems you see with groovy?

**Comment by nik9000** [comment]
> Can you elaborate what problems you see with groovy?

@breskeby and I met over video so I could show him the errors in eclipse - basically it doesn't pick up the right version of groovy.

**Pull request merged**